src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -82,27 +82,35 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
     </OakLessonBottomNav>
   );
   const handleShareResultsClick = async () => {
+    setIsAttemptingShare(true);
```

Comment:
Can we have a test for the log and error handling ?

Reply:
there is a test for that pupilReview.view.test.tsx suite